src/toil/test/docs/scripts/tutorial_jobfunctions.py

```diff
@@ -0,0 +1,15 @@
+from toil.common import Toil
```

Comment:
In the context of the tutorial, I believe it makes sense to keep these separate even though they are very similar because the tutorial is emphasizing the small difference between them:

"Here helloWorld() is a job function. It uses the toil.job.Job.log() to log a message that will be printed to the output console. Here the only subtle difference to note is the line:
hello_job = Job.wrapJobFn(helloWorld, "Woot")
Which uses the function toil.job.Job.wrapJobFn() to wrap the job function instead of toil.job.Job.wrapFn() which wraps a vanilla function."

Reply:
Alright, if you feel it's a big enough difference, then I'm good with it.